web/modules/custom/pineapple_helper/templates/raise-issue-dialog.html.twig

```diff
@@ -0,0 +1,5 @@
+<!-- Add you custom twig html here -->
+<p>
+  <strong>Report an Issue</strong>
+</p>
+<a href="/contact" class="button button--primary" target="_blank">Report an issue</a>
```

Comment:
Add more content which is helpful for the user.

Reply:
Done